docs/en/latest/plugins/sls-logger.md

```diff
@@ -23,32 +29,29 @@ title: sls-logger
-
-This plugin provides the ability to push Log data as a batch to ali cloud log service. In case if you did not receive the log data don't worry give it some time it will automatically send the logs after the timer function expires in our Batch Processor.
+The `sls-logger` Plugin is used to push logs to [Ali Cloud log server](https://help.aliyun.com/document_detail/112903.html?spm=a2c4g.11186623.6.763.21321b47wcwt1u) using [RF5424](https://tools.ietf.org/html/rfc5424).
 
-For more info on Batch-Processor in Apache APISIX please refer
-[Batch-Processor](../batch-processor.md)
+It might take some time to receive the log data. It will be automatically sent after the timer function in the [batch processor](../batch-processor.md) expires.
 
 ## Attributes
 
-|Name           |Requirement    |Description|
-|---------      |--------       |-----------|
-|host           |required       | IP address or the Hostname of the TCP server, please reference ali cloud log [Serve List](https://help.aliyun.com/document_detail/29008.html?spm=a2c4g.11186623.2.14.49301b4793uX0z#reference-wgx-pwq-zdb), use IP address instead of domain.|
-|port           |required       |Target upstream port, default 10009.|
-|timeout        |optional       |Timeout for the upstream to send data.|
-| project |required|Ali cloud log service project name，please create in sls before us this plugin.|
-| logstore | required |Ali cloud log service  logstore name，please create in sls before us this plugin.|
-| access_key_id | required | Ali cloud AccessKey ID, reference [Authorization](https://help.aliyun.com/document_detail/47664.html?spm=a2c4g.11186623.2.15.49301b47lfvxXP#task-xsk-ttc-ry).|
-| access_key_secret | required |Ali cloud AccessKey Secret, reference [Authorization](https://help.aliyun.com/document_detail/47664.html?spm=a2c4g.11186623.2.15.49301b47lfvxXP#task-xsk-ttc-ry).|
-| include_req_body | required| Boolean value. |
-|name           |optional       |A unique identifier to identity the batch processor.|
+| Name              | Required | Description                                                                                                                                                                                                                                     |
+|-------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| host              | True     | IP address or the hostname of the TCP server. See [Ali Cloud log documentation](https://help.aliyun.com/document_detail/29008.html?spm=a2c4g.11186623.2.14.49301b4793uX0z#reference-wgx-pwq-zdb) for details. Use IP address instead of domain. |
```

Comment:
```suggestion
| host              | True     | IP address or the hostname of the TCP server. See [Alibaba Cloud Log Service](https://www.alibabacloud.com/help/en/log-service/latest/endpoints) for details. Use IP address instead of domain. |
```

Reply:
We should use English documentation links.